app/src/main/res/values-es/strings.xml

```diff
@@ -1,110 +1,115 @@
+    <string name="label_speaker_autosave">Guardar el volumen automáticamente</string>
+    <string name="description_speaker_autosave">La entrada \"Altavoz del dispositivo\", que es responsable de los volúmenes predeterminados, se actualiza cuando se conecta un dispositivo Bluetooth. Nota: Esto no funciona en todos los dispositivos.</string>
     <string name="action_show">Mostrar</string>
-    <string name="action_dismiss">Ignorar</string>
-    <string name="battery_optimizations_issue_description">Las optimizaciones de la batería están habilitadas para esta aplicación y pueden evitar que reciba eventos Bluetooth. Considere desactivar las optimizaciones si está experimentando problemas.</string>
-    <string name="label_monitoring_duration">Monitor De Duration</string>
-    <string name="description_monitoring_duration">Después de que haya echo ajustes, habrá una duración en milisegundos, en cual todo tipo de ajustes de volumen seria bloqueados.</string>
+    <string name="action_dismiss">Descartar</string>
+    <string name="battery_optimizations_issue_description">Las optimizaciones de batería están habilitadas para esta aplicación y pueden evitar que reciba eventos de Bluetooth. Considere deshabilitar las optimizaciones si tiene problemas.</string>
+    <string name="label_monitoring_duration">Duración del monitoreo</string>
+    <string name="description_monitoring_duration">Una duración en milisegundos, después de que se hayan realizado los ajustes, durante la cual se bloquearán los cambios de volumen adicionales.</string>
     <string name="label_volume_lock">Bloqueo de volumen</string>
-    <string name="description_volume_lock">Cualquier cambio de volumen qué no se haya echo por esta aplicación sera revertido mientras el deposito esté conectado.</string>
-    <string name="label_bluetooth_settings">Ajustes de Bluetooth</string>
-    <string name="label_keep_awake">Mantener despierto</string>
-    <string name="description_keep_awake">Mantener pantalla encendida mientras este conectado el dispositivo.</string>
+    <string name="description_volume_lock">Cualquier cambio de volumen no realizado a través de esta aplicación se revertirá mientras este dispositivo esté conectado.</string>
+    <string name="label_bluetooth_settings">Configuración del Bluetooth</string>
+    <string name="label_keep_awake">Mantente activo</string>
+    <string name="description_keep_awake">Mantener la pantalla encendida mientras este dispositivo está conectado.</string>
+    <string name="android10_applaunch_issue_description">Android 10 restringe cómo se pueden iniciar las aplicaciones desde el fondo. Para permitir la apertura de aplicaciones cuando se conecta un dispositivo Bluetooth, otorgue el permiso "Mostrar sobre otras aplicaciones" (SYSTEM_ALERT_WINDOW).</entry>
```

Comment:
`</entry>` should be `</string>`

Reply:
Here I accidentally and without knowing why I changed it.

But you are right. Sorry for this mistake.

Best regards